subcommands/common_config.go

```diff
@@ -226,16 +226,13 @@ func SetUpdatesConfig(opts *SetUpdatesConfigOptions, reportedTag string, reporte
 	if len(configuredTag) == 0 && len(reportedTag) > 0 {
 		configuredTag = reportedTag
 	}
-	if len(configuredApps) == 0 && reportedApps != nil {
-		configuredApps = strings.Join(reportedApps, ",")
-	}
```

Comment:
I believe that the `reportedTag` is subject to the same problem as `reportedApps`.
So, it'd be wise to update that logic accordingly as well.

Reply:
Done. :+1:

Reply:
note - this is rolling back a "fix" - https://github.com/foundriesio/fioctl/commit/a8a21c70d9dfda5b2218bc93af0f52a6d6fe6508

Reply:
or is it. i'm confused. can you add a little more to the commit message?

Reply:
@doanac IIUC that fix is nice, but it introduced another bug we were discussing today.
So, Eric has now moved it to a better place (see lines below).

Reply:
> note - this is rolling back a "fix" - [a8a21c7](https://github.com/foundriesio/fioctl/commit/a8a21c70d9dfda5b2218bc93af0f52a6d6fe6508)

I think that issue can be fixed by making `fioconfig` send the config to DG. It becomes possible with this change https://github.com/foundriesio/fioconfig/pull/45.